src/metabase/actions/execution.clj

```diff
@@ -159,11 +156,8 @@
 
                   (= implicit-action :row/update)
                   (assoc :update-row row-parameters))]
-    (try
-      (binding [qp.perms/*card-id* (:model_id action)]
-        (actions/perform-action! implicit-action arg-map))
-      (catch Exception e
-        (handle-action-execution-error e)))))
+    (binding [qp.perms/*card-id* (:model_id action)]
+      (actions/perform-action! implicit-action arg-map))))
```

Comment:
I think we need a nice job catching and rethrowing errors inside `perform-action!`, there is no need to do another try catch here.